drivers/usb-common.c

```diff
@@ -361,18 +374,23 @@ static int match_function_regex(USBDevice_t *hd, void *privdata)
 		    __func__, "Bus", hd->Bus);
 		return r;
 	}
+
+	r = match_regex(data->regex[6], hd->Device);
+	if (r != 1) {
+		return r;
```

Comment:
To be consistent with other code around, can you please add the `upsdebugx()` call for the failure, probably similar to one you added at https://github.com/networkupstools/nut/pull/974/files#diff-6aea68febe9f0fc9ad47c0d821a43073c65461e02ca9f0648ba78981482f98b8R122 ?

Otherwise, looks good to me.